docs/data/xrootd/install-gateway.md

```diff
@@ -0,0 +1,188 @@
+Install XRootD Gateway
+======================
+
+[XRootD](http://xrootd.org/) is a hierarchical storage system that can be used in a variety of ways to access data,
+typically distributed among actual storage resources. In this document we focus on using XRootD as a simple layer
+exporting an underlying storage system (e.g., [HDFS](/data/install-hadoop.md)) to the outside world.
+
+!!! note
+    The OSG only supports XRootD gateway installations on EL7 hosts
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points:
+
+-   **User IDs:** If it does not exist already, the installation will create the Linux user ID `xrootd`
+-   **Service certificate:** The XRootD service uses a host certificate and key pair at
+    `/etc/grid-security/xrd/xrdcert.pem` and `/etc/grid-security/xrd/xrdkey.pem`
+-   **Networking:** The XRootD service uses port 1094 by default
+
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+
+-   Ensure the host has [a supported operating system](/release/supported_platforms)
+-   Obtain root access to the host
+-   Prepare [the required Yum repositories](/common/yum)
+-   Install [CA certificates](/common/ca)
+
+Installing XRootD
+-----------------
+
+To install XRootD, run the following Yum command:
+
+``` console
+root@xrootd-server # yum install xrootd
+```
+
+Configuring XRootD
+------------------
+
+You will need to modify the file `/etc/xrootd/xrootd-server.cfg` that we will refer now on (unless explicitly said) as THE configuration file.
+A simple example of such a configuration is below with further customizations in the rest of the document.
+
+``` file
+xrd.port 1094
+all.role server
+# Allow all contents of /store to be served from this host
+all.export %RED%/store/%ENDCOLOR% readonly
+
+cms.allow host *
+# Logging verbosity
+xrootd.trace emsg login stall redirect
+ofs.trace -all
+xrd.trace conn
+cms.trace all
+
+all.adminpath /var/run/xrootd
+all.pidpath   /var/run/xrootd
+# This site name is only used for monitoring purposes (ex: T2_US_UCSD or UCSD)
+all.sitename   %RED%YOUR SITE NAME%ENDCOLOR%
+
+xrd.report     xrootd.t2.ucsd.edu:9931,desire.physics.ucsd.edu:9931 every 30s all sync
+xrootd.monitor all auth flush io 60s ident 5m mbuff 8k rbuff 4k rnums 3 window 10s dest files io info user redir xrootd.t2.ucsd.edu:9930 dest files iov info user xrootd.t2.ucsd.edu:9932
+
+xrd.network keepalive kaparms 10m,1m,5
+xrd.timeout idle 60m
+```
+
+Notice the following:
+
+1. The directory `/store` is the one your users will interact "see" from this xrootd gateway. Other control options can be added like "writable" see [documenation here](http://xrootd.org/doc/dev48/ofs_config.htm).
```

Comment:
Is there a particular part of this manual that we want to point out? Because just linking the entire manual is pretty user hostile. It'd be like linking the index of the condor manual and telling them that they can read it to learn about security configuration.

Also you have a typo here, "documenation" (please run a spell checker)